pkgs/os-specific/linux/wooting-udev-rules/default.nix

```
@@ -2,9 +2,9 @@
 
 stdenv.mkDerivation rec {
   pname = "wooting-udev-rules";
-  version = "20210525";
+  version = "unstable-20230331";
```

```suggestion
  version = "unstable-2023-03-31";
```
for dates we use iso format

---

Ah yes. I missed that first time round. Fixed.